modules/installation-osp-provider-network-preparation.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included
+// TODO: Add mods to all appropriate installation assemblies after first pass is approved.
+
+[id="installation-osp-provider-network-preparation_{context}"]
+= {rh-openstack} provider network requirements for cluster installation
+
+Before you install a {product-title} cluster, your {rh-openstack-first} deployment and provider network must meet a number of conditions:
+
+* The link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/networking_guide/sec-networking-concepts#install-networking[{rh-openstack} networking service (Neutron) is enabled] and accessible through the {rh-openstack} networking API.
+* The {rh-openstack} networking service has the link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/networking_guide/sec-allowed-address-pairs[port security and allowed address pairs extensions enabled].
+* The provider network can be shared with other tenants.
++
+[TIP]
+====
+Use the `openstack network create` command with the `--share` flag to create a network that can be shared.
+====
+* The {rh-openstack} project that you use to install the cluster must own the provider network, as well as an appropriate subnet.
++
+[TIP]
+====
+To create a network for a project that is named "openshift," on a command line, enter:
+[source,terminal]
+----
+$ openstack network create --project openshift
+----
+
+To create a subnet for a project that is named "openshift," on a command line, enter:
+[source,terminal]
+----
+$ openstack subnet create --project openshift
+----
+
+To learn more about creating networks on {rh-openstack}, read link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/networking_guide/sec-networking-concepts#provider-networks[the provider networks documentation].
+====
++
+If the cluster is owned by the `admin` user, you must run the installer as that user to create ports on the network.
++
+[IMPORTANT]
+====
+// FIXME: Should we even extend this as an option, then?
+If the network and subnet are owned by an administrator, Terraform will fail to create ports during installation. The ports must be created manually.
```

Comment:
@EmilienM Any thoughts on whether or not this line should be included? My inclination is to not dither about it and just say must, i.e. L38.

Reply:
I'll come back to it tomorrow.

Reply:
I would phrase it like:
```
Provider networks must be owned by the OpenStack project that is used to create the OpenShift cluster. Otherwise Nova won’t be able to request a port from that network (since it’s a network of “external” type).
```